### DISCUSSION TITLE
Switching from zzh! to sonoff 3.0

### DISCUSSION THREAD

**Discussion by pannal** [Koenkk/zigbee2mqtt#14014] [Devices]
Hey,

I've been successfully using an Electrolama zzh for years (CC2652R). After a little hickup, which was ultimately resolved by disconnecting the stick for a while, I got anxious and thought I'd like to have a backup coordinator.

So I bought a SONOFF Zigbee 3.0 USB Dongle Plus (CC2652P+CP2102N).

Should the dreaded day come that none of my lights work anymore without my zzh, can I switch over seamlessly to the backup coordinator dongle?

Thanks!

**Comment by sjorge** [comment]
As long as they are running the same firmware and you have a succesfully create coordinator_backup.json z2m should be able to restore the coordinator state to the stick.

**Reply by pannal** [reply]
Thanks! Is there a way to do that automatically?
